doc/integrator/make.rst

```diff
@@ -0,0 +1,83 @@
+.. _integrator_make:
+
+Build the project with make
+===========================
+
+Makefiles
+---------
+
+Usually we have the following makefiles includes:
+``<user>.mk`` -> ``<package>.mk`` -> ``CONST_Makefile.mk``.
+
+The ``CONST_Makefile.mk`` is a huge makefile that maintain by the
+c2cgeoportal developers.
+
+The ``<package>.mk`` contains the project specific things,
+by default if contains:
+
+.. code:: makefile
+
+    ifdef VARS_FILE
+    VARS_FILES += ${VARS_FILE} vars_<package>.yaml
+    else
+    VARS_FILE = vars_<package>.yaml
+    VARS_FILES += ${VARS_FILE} 
+    endif
+
+    include CONST_Makefile
+
+
+And the ``<user>.mk`` contains the user specific things, mainly the
```

Comment:
``<user>.mk`` contains the user-specific config (mainly the ``INSTANCE_ID`` config) and should look like: